composer.json

```diff
@@ -19,6 +19,7 @@
         "php": ">=5.3.3",
         "rych/hash_pbkdf2-compat": "~1.0",
         "ext-gmp": "*",
+        "ext-bcmatch": "*",
```

Comment:
I knew I saw something with this before - can you change this to bcmath? :) the tests failed because of this